docs/templates/configuration.md.yml

```diff
@@ -75,6 +75,22 @@ configOptions:
           // This key will be extracted
           myT("key0");
           ```
+  - name: transComponentNames
+    type: "{sourceModule: string; importName: string}[]"
+    description: Possible import names of `Trans` components. Note that, for now, it does a pretty dumb job at resolving the source module so you should list all ways you have to import your components.
+    defaultValue: "[{sourceModule: 'react-i18next', importName: 'Trans'}]"
```

Comment:
Did you choose this API for forward-compatibility purpose? I feel like the following API would be more comfortable to use:
```javascript
[
  ['react-i18next', 'Trans'],
  ['./src/components/CustomTransComponent', 'Trans'],
]
```
For forward-compatibility, a third value could be added to the array if it's needed someday. WDYT?

Reply:
Nope, I did that mostly to be as explicit as possible. I've switched to your way.